src/backend/gl/src/info.rs

```diff
@@ -190,6 +192,41 @@ pub struct Info {
     pub extensions: HashSet<&'static str>,
 }
 
+bitflags! {
+    pub struct LegacyFeatures: u16 {
```

Comment:
Do you want to keep those in GL? If someone wants the target all backends, they need to have access to these flags. We might have dx11 specific flags also.

Reply:
I want to keep it in GL for now, then we'll think about expressing them in an extension to Vk portability, at which point we'll need to move them back into HAL.